pkg/xen/Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN [ -f "$(basename ${XEN_SOURCE})" ] || tar --absolute-names -xz < /xen.tar.gz
 WORKDIR /xen/xen
 COPY *.patch arch /tmp/
 RUN cp /tmp/"$(uname -m)"/*.patch /tmp/
-RUN for p in /tmp/*.patch ; do patch -p1 < "$p" || exit 1 ; done
+#RUN for p in /tmp/*.patch ; do patch -p1 < "$p" || exit 1 ; done
```

Comment:
If we don't need the patch anymore, just remove this line and the deprecated patch file....